### DISCUSSION TITLE
How to get Selenium and webbrowser with MicroPython ?

### DISCUSSION THREAD

**Discussion by Remione** [micropython/micropython#13070] [Libraries & Drivers]
Morning,

i wrote a script to connect to a website, take screenshot and send them through a TL Bot, using Selenium and Webbrowser. On computer is working the code perfectly. But of course, when I need to transfer it to my ESP32 (espressif WROOM 32), it's missing the librairies Selenium and Webbrowser.

I included mip to import them but they are not in MicroPython libraries.

Question:
how can I import them in MicroPython ?

```py
Connection successful
Installing webbrowser (latest) from https://micropython.org/pi/v2 to /lib
Package not found: https://micropython.org/pi/v2/package/6/webbrowser/latest.json
Package may be partially installed
Installing selenium (latest) from https://micropython.org/pi/v2 to /lib
Package not found: https://micropython.org/pi/v2/package/6/selenium/latest.json
Package may be partially installed
Traceback (most recent call last):
  File "<stdin>", line 23, in <module>
ImportError: no module named 'webbrowser' 
```

Thank you !

**Comment by Josverl** [comment]
These libraries are not available for MicoPython.

It's unlikely they could be ported.
